src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+import { HStack, Flex } from "@chakra-ui/react"
+
+export interface FooterProps {
+  children: JSX.Element
+}
+
+export const Footer = ({ children }: FooterProps): JSX.Element => (
+  <Flex
+    h="80px"
```

Comment:
btw, we should specify this in rem probably. this is because rem is actually dynamically computed relative to the root element and allows responsive sizing (see [here](https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/))